demo/src/app/components/dropdown/demos/navbar/dropdown-navbar.html

```diff
@@ -0,0 +1,47 @@
+<p>
+  Custom placement of a dropdown in a navbar is only possible if the <code>display</code> property is explicitly
```

Comment:
Even though the demo title is explicitly `Dynamic positioning in a navbar` I would start this documentation section by saying something like:

> By design dropdowns are always positioned dynamically via [placement](/positioning#dropdown) expect when used inside navbar elements. To gracefully display them properly on small screens, they are rendered in the html as block elements.

Then I would continue with your `Custom placement of a dropdown ...` and finally I would remove ~`Beware however that this breaks the positioning of the dropdown when the navbar is expanded on small displays`~